client/post-editor/editor-ground-control/index.jsx

```diff
@@ -35,9 +37,12 @@ export default React.createClass( {
 		onPreview: React.PropTypes.func,
 		onPublish: React.PropTypes.func,
 		onSaveDraft: React.PropTypes.func,
+		onMoreInfoAboutEmailVerify: React.PropTypes.func,
 		post: React.PropTypes.object,
 		savedPost: React.PropTypes.object,
 		site: React.PropTypes.object,
+		user: React.PropTypes.object,
+		userUtils: React.PropTypes.object,
```

Comment:
Further, can we do the same with `user`? As much as possible, avoid passing props from ancestors down to their children if the ancestors don't necessarily need to care about the concerns of that descendent component's behavior.